src/audio-player-composable/components/seek-bar/seek-bar.tsx

```diff
@@ -0,0 +1,147 @@
+import React, {useCallback, useContext} from 'react';
+import {getTrackBackground} from 'react-range';
+import {AudioPlayerContext} from '../../context';
+import {SeekBarProps} from './types';
+import {Slider, SliderProps} from '../../../slider';
+import {ScreenReaderOnly} from '../../../screen-reader-only';
+import {useReactKeys} from '../../../utils/hooks';
+import {filterOutFalsyProperties} from '../../../utils/filter-object';
+import {getToken} from '../../../utils/get-token';
+import {getSingleStylePreset} from '../../../utils';
+import {StyledTrack} from '../../../slider/styled';
+import {withOwnTheme} from '../../../utils/with-own-theme';
+import stylePresets from './style-presets';
+import {useTheme} from '../../../theme';
+import {formatTrackData, seekBarAriaValueText} from '../../utils';
+import defaults from './defaults';
+
+const ThemelessSeekBar: React.FC<SeekBarProps> = ({overrides = {}}) => {
+  const {slider: seekBarSliderOverrides} = overrides;
+
+  const theme = useTheme();
+  const {
+    seekBar: {slider: seekBarSliderDefaults},
+  } = theme.componentDefaults;
+
+  const {getSeekBarProps} = useContext(AudioPlayerContext);
```

Comment:
can we create a separate hook that returns the context?

in the context.tsx file
```
export const useAudioPlayerContext = useContext(AudioPlayerContext);
```

```suggestion
  const {getSeekBarProps} = useAudioPlayerContext();
```

it's no big deal but that makes things a bit neater 

ps, update the PlayPause component to use the hook